### PR TITLE
N3LO benchmark study

### PR DESCRIPTION
**Pull request by giacomomagni**
This PR is to keep track of the scripts needed for the N3LO benchmark study.

### CODE REVIEW

File: extras/lh_bench_23/parse_to_latex.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        for the initial conditions and the input parton distributions
+        given in Sec.~\ref{sec:toy_pdf},
+        with the FHMRUVV splitting functions approximation.
+    """
+    dump_table(scheme, approx, caption)
+
+    approx = "EKO"
+    scheme = "FFNS"
+    caption = r"""
+        Same as \ref{tab:n3lo_ffns_fhmruvv} but now with
```
Comment (felixhekhorn) [inline extras/lh_bench_23/parse_to_latex.py:151]:
```suggestion
        Same as \cref{tab:n3lo_ffns_fhmruvv} but now with
```

File: extras/lh_bench_23/parse_to_latex.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        the NNPDF splitting functions approximation.
+    """
+    dump_table(scheme, approx, caption)
+
+    approx = "EKO"
+    scheme = "VFNS"
+    caption = r"""
+        Same as \ref{tab:n3lo_vfns_fhmruvv} but now with
```
Comment (felixhekhorn) [inline extras/lh_bench_23/parse_to_latex.py:159]:
```suggestion
        Same as \cref{tab:n3lo_vfns_fhmruvv} but now with
```

File: extras/lh_bench_23/parse_to_latex.py [hunk 3]
```diff
@@ -26,6 +26,28 @@
...
+    \multicolumn{1}{c|} {$xL_-$} &
+    \multicolumn{1}{c|} {$xL_+$} &
+    \multicolumn{1}{c|} {$xs_v$} &
+    \multicolumn{1}{c|} {$xs_+$} &
+    \multicolumn{1}{c|} {$xc_+$} &
+    \multicolumn{1}{c||}{$xg$}
+    """
+
 
 def insert_haedrule(scheme, approx, caption):
```
Comment (felixhekhorn) [inline extras/lh_bench_23/parse_to_latex.py:52]: `insert_haedrule` -> `insert_headrule` :upside_down_face:

Comment (giacomomagni) [reply extras/lh_bench_23/parse_to_latex.py]: done in 753475ab

**Comment by giacomomagni** [comment]
@felixhekhorn given that the paper has been published shall we merge this?

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**